### ISSUE TITLE
Monthly budget

### ISSUE THREAD

**Issue by danielyrovas** [danilkinkin/buckwheat#43]
I would like the application to have an automatically rolling monthly budget that shows how much you have used out of the total that you set for each month (rather than daily display for the time period)

I _really_ like the interface of the app, very clean.
Have you got a matrix group for the project?

**Comment by aqilc** [comment]
May I add to this, that maybe include a weekly budget too? I don't get paid monthly, only biweekly and only some weeks, so I would much prefer a weekly one!

**Comment by LuminarySage** [comment]
Yes I'd love the ability to have different budgeting periods instead of being stuck to a daily budget. I feel like most people budget weekly or monthly so it's a sorely needed feature
